.github/workflows/ci.yml

```diff
@@ -3,26 +3,42 @@ on:
   pull_request:
     branches: [main]
 jobs:
-  check-conventional-commits:
+  conventional-commits:
     runs-on: ubuntu-latest
```

Comment:
it is very bad define every steps in separate job because then you consume lot of resources - and starting time of environment take 90-99% of relevant step - PLS merge related step to job - for inspiration you can check: https://github.com/ohpen/shared-java-workflow-template